lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -276,6 +277,19 @@ def request_three_d_secure(post, options = {})
         post[:payment_method_options][:card][:request_three_d_secure] = options[:request_three_d_secure]
       end
 
+      def add_external_three_d_secure_auth_data(post, options = {})
+        return unless options[:three_d_secure]&.is_a?(Hash)
+
+        three_d_secure = options[:three_d_secure]
+        post[:payment_method_options] ||= {}
+        post[:payment_method_options][:card] ||= {}
+        post[:payment_method_options][:card][:three_d_secure] ||= {}
+        post[:payment_method_options][:card][:three_d_secure][:version] = three_d_secure[:version] || (three_d_secure[:ds_transaction_id] ? '2.1.0' : '1.0.2')
```

Comment:
just curious, but how do we determine the minor version here (e.g. `2.1.0` rather than `2.2.0`)

Reply:
Unfortunately I don't think we can, and Stripe requires that a minor version is specified, so this fallback is hard coded. If happy to bump it if you think 2.2.0 is more common, but figured lower is safer.

Reply:
That makes sense, was just wondering! The lower version you have now seems like a good starting point to me too 👍

Reply:
I see now that exemptions are the primary change in 2.2.0. 🤔  Perhaps it's it better to side with 2.2.0, given possible discrepancy?

Reply:
hmmm, maybe we try 2.2.0 to start and then just update if we notice errors?

Reply:
I guess really for our purposes, we will most likely always have a version directly passed in